tests/unittest/_torch/attention/sparse/deepseek_v4/test_deepseek_v4_indices_transform.py

```
@@ -0,0 +1,540 @@
+
+                    physical_idx = _local_to_physical_idx(
+                        local_idx, block_tables_compressed[r], tokens_per_block_compressed
+                    )
+                    expected = compressed_buffer_flat[physical_idx]
+
+                    torch.testing.assert_close(
+                        actual, expected, msg=f"Token {t} compress out[{out_pos}]: value mismatch"
+                    )
+
+    # Cleanup
+    for req, ctx_len in zip(requests, context_lengths):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)